### PR TITLE
schema: planning/budget/project: remove guidance on translations

### LINKED ISSUE #1485: Update `planning/budget/project` description
The current description for the `planning/budget/project` field is:

> The name of the project through which this contracting process is funded (if applicable). Some organizations maintain a registry of projects, and the data should use the name by which the project is known in that registry. No translation option is offered for this string, as translated values can be provided in third-party data, **linked from the data source above**.

However, the `planning/budget/source` field was deprecated in 1.1.

### PR DESCRIPTION
**Pull request by yolile**
closes #1485

### CODE REVIEW

File: docs/history/changelog.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1353](https://github.com/open-contracting/standard/pull/1353) Replace "tender submissions" with "bid submissions".
   * [#1401](https://github.com/open-contracting/standard/pull/1401) Replace "total value" with "value".
   * [#1403](https://github.com/open-contracting/standard/pull/1403) Replace "contract notice" with "contract signature notice".
+  * [](https://github.com/open-contracting/standard/pull/1487) `planning.budget.project`, to remove no translation option rationale.
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
  * [#1487](https://github.com/open-contracting/standard/pull/1487) Remove sentence about translation options from `planning.budget.project`.
```

Comment (yolile) [reply docs/history/changelog.md]: I followed the same pattern as the first bullet point here "[#1109](https://github.com/open-contracting/standard/pull/1109) `tender.awardCriteriaDetails`, to remove "selection criteria"." should we change that one?

Comment (jpmckinney) [reply docs/history/changelog.md]: Good catch

```
`planning.budget.project`, to remove sentence about translation options.
```

Comment (yolile) [reply docs/history/changelog.md]: And maybe I should move it below (or above?) that one to keep all the ones related to specific fields together?

Comment (jpmckinney) [reply docs/history/changelog.md]: Yes, good call

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**